test/smoke/testApps/SpringScheduler/src/main/java/com/microsoft/ajl/simple/SpringSchedulerApp.java

```diff
@@ -23,12 +23,6 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder applicatio
 
     @Scheduled(fixedRate = 100)
     public int fixedRateScheduler() throws IOException {
```

Comment:
just to avoid confusion later, returning `void` here and performing `System.out.println("scheduled job run")` in the body is clearer i think